src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.ConflictResolution.targets

```diff
@@ -32,10 +32,14 @@ Copyright (c) .NET Foundation. All rights reserved.
 
       
       <!-- Also include RuntimeTarget items, which aren't included in AllCopyLocalItems, but need to be considered
-           for conflict resolution -->
+           for conflict resolution.  Set DestinationSubPath for these items so that conflict resolution will consider
```

Comment:
Don't we need some sort of condition on this for frameworks that don't understand RuntimeTarget items, or did you have you decided that you're adding the RuntimeTarget concept to desktop?

Reply:
This just tries to match the path that will come out of PublishAssembliesResolver here when doing conflict resolution.  If you have a package that assumes that native assets under `runtimes/` will be loaded when used in an AnyCPU .NET Framework app, then those assets will be placed in the `runtimes/` subfolders when the app is published and will fail to load at runtime.  This doesn't change that.

If you publish for a specific RID, then the items will be `runtime` items instead of `runtimeTargets` items and will get copied directly to your output folder, and conflict resolution will use the right path.  For example, under the `.NETCoreApp,Version=v2.0` target in an assets file:

```json
"System.Diagnostics.TraceSource/4.0.0": {
  "type": "package",
  "dependencies": {
    "Microsoft.NETCore.Platforms": "1.0.1",
    "System.Collections": "4.0.11",
    "System.Diagnostics.Debug": "4.0.11",
    "System.Globalization": "4.0.11",
    "System.Resources.ResourceManager": "4.0.1",
    "System.Runtime": "4.1.0",
    "System.Runtime.Extensions": "4.1.0",
    "System.Threading": "4.0.11",
    "runtime.native.System": "4.0.0"
  },
  "compile": {
    "ref/netstandard1.3/System.Diagnostics.TraceSource.dll": {}
  },
  "runtimeTargets": {
    "runtimes/unix/lib/netstandard1.3/System.Diagnostics.TraceSource.dll": {
      "assetType": "runtime",
      "rid": "unix"
    },
    "runtimes/win/lib/netstandard1.3/System.Diagnostics.TraceSource.dll": {
      "assetType": "runtime",
      "rid": "win"
    }
  }
},
```

While for the `.NETCoreApp,Version=v2.0/win10-x64` target, it looks like this:

```json
"System.Diagnostics.TraceSource/4.0.0": {
  "type": "package",
  "dependencies": {
    "Microsoft.NETCore.Platforms": "1.0.1",
    "System.Collections": "4.0.11",
    "System.Diagnostics.Debug": "4.0.11",
    "System.Globalization": "4.0.11",
    "System.Resources.ResourceManager": "4.0.1",
    "System.Runtime": "4.1.0",
    "System.Runtime.Extensions": "4.1.0",
    "System.Threading": "4.0.11",
    "runtime.native.System": "4.0.0"
  },
  "compile": {
    "ref/netstandard1.3/System.Diagnostics.TraceSource.dll": {}
  },
  "runtime": {
    "runtimes/win/lib/netstandard1.3/System.Diagnostics.TraceSource.dll": {}
  }
},
```

Reply:
I understand, I'm talking about the desktop case where runtimeTargets are meaningless since nothing knows how to load them.